docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -0,0 +1,453 @@
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0,1)
+qc.draw("mpl")
+```
+
+![Initial QuantumCircuit](/images/verify/use_estimator/initialize.png)
+
+<Admonition type="note" title="Note">
+  The [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) you pass to [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) must not include any measurements.
+</Admonition>
+
+### Initialize `Estimator`
+
+Next, instantiate an [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).
+
+```python
+from qiskit.primitives import Estimator
+
+estimator = Estimator()
+```
+
+### Run and get results
+
+Now that you have defined your `estimator`, you can run your estimation by calling the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method,
+which returns an instance of [`qiskit.providers.JobV1`](../api/qiskit/qiskit.providers.JobV1). You can get the results from the job (as a [`qiskit.primitives.EstimatorResult`](../api/qiskit/qiskit.primitives.EstimatorResult) object)
+with the [`qiskit.providers.JobV1.result`](../api/qiskit/qiskit.providers.JobV1#result) method.
+
+```python
+job = estimator.run(qc, observable)
+result = job.result()
+print(result)
+```
+
+```python
+EstimatorResult(values=array([4.]), metadata=[{}])
+```
+
+This example only uses one [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) and one observable. If you want to get expectation values for multiple circuits and observables, you can pass a `list` of [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit)s and a list of `BaseOperator`s to the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method. Both `list`s must have the same length.
+
+#### Get the expected value
+
+From these results you can extract the expected values with the attribute [`qiskit.primitives.EstimatorResult.values`](../api/qiskit/qiskit.primitives.EstimatorResult#values).
+
+[`qiskit.primitives.EstimatorResult.values`](../api/qiskit/qiskit.primitives.EstimatorResult#values) returns a `numpy.ndarray`
+whose `i`<sup>th</sup> element is the expectation value corresponding to the `i`<sup>th</sup> circuit and `i`<sup>th</sup> observable.
+
+```python
+exp_value = result.values[0]
+print(exp_value)
+```
+
+```python
+3.999999999999999
+```
+
+### Parameterized circuit with `Estimator`
+
+The [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) primitive can be run with unbound parameterized circuits like the one below.
+You can also manually bind values to the parameters of the circuit and follow the steps of the previous example.
+
+```python
+from qiskit.circuit import Parameter
+
+theta = Parameter('θ')
+param_qc = QuantumCircuit(2)
+param_qc.ry(theta, 0)
+param_qc.cx(0,1)
+print(param_qc.draw())
+```
+
+```
+    ┌───────┐
+q_0: ┤ Ry(θ) ├──■──
+    └───────┘┌─┴─┐
+q_1: ─────────┤ X ├
+            └───┘
+```
```

Comment:
The rendering of this drawing seems funky. I believe it used to look more aligned, in its original state. Any thoughts on how to fix the formatting? (Same applies to the drawing in the Sampler section below)
<img width="236" alt="Screenshot 2023-10-17 at 12 53 17 PM" src="https://github.com/Qiskit/documentation/assets/28769477/fdfce899-96c8-463e-a7bd-427f27545054">